app/models/book.rb

```diff
@@ -1,2 +1,6 @@
 class Book < ActiveRecord::Base
+  monetize :price_cents
+
+  scope :filter, -> (search) { where("title LIKE ?", "%#{search}%") } 
```

Comment:
I know I brought this up previously when we spoke but THIS is the correct way to do a like query in Rails :+1: